style.css

```
@@ -603,6 +570,248 @@ body[data-bg="wallpaper"] .tooltip-text {
+
+.gapps-edit-btn:active {
+	transform: scale(0.95);
+}
+
+.gapps-edit-btn.active {
+	background-color: var(--darkerColor-blue);
+	box-shadow: 0 0 0 2px var(--accentLightTint-blue);
+}
+
+.gapps-edit-btn svg {
+	fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)